ecs-cli/modules/utils/regcreds/creds_input_reader.go

```diff
@@ -74,6 +74,22 @@ func ReadCredsInput(filename string) (*ECSRegCredsInput, error) {
 	return credsInput, nil
 }
 
+// ReadCredsOutput parses an ecs creds output file into an RegistryCredsOutput struct
+// TODO: use this to parse reg creds used with "compose" cmd
+func ReadCredsOutput(filename string) (*RegistryCredsOutput, error) {
```

Comment:
Suggestion about the organization of this package, `readers`: One is that I notice that you also have something that writes the cred output file, so "readers" might not be the best name. I think it would make more organizational sense to have one file `types.go` that contains the structs, another file `readers.go` that containers the reading code, and a third file `writers.go` for the writer. Then may be the whole package could be called `credfiles` or something. Just a suggestion.

Reply:
Yeah, it's a bit of a misnomer now. Also, I like the idea of splitting the types (and their methods) out from the functions.